solidity/deploy/09_transfer_proxy_admin_ownership.ts

```diff
@@ -0,0 +1,23 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { ethers, getNamedAccounts, upgrades, deployments } = hre
+  const { keepTechnicalWalletTeam } = await getNamedAccounts()
+  const { deployer } = await ethers.getNamedSigners()
+
+  // TODO: Once a DAO is established we want to switch to ProxyAdminWithDeputy and
+  // use the DAO as the proxy admin owner and ESDM as the deputy. Until then we
+  // use governance as the owner of ProxyAdmin contract.
+  const newProxyAdminOwner = keepTechnicalWalletTeam
```

Comment:
Let's use ESDM, just like [for the `WalletRegistry`](https://github.com/keep-network/keep-core/blob/5dd438efeeef4e5f71a264304f810786607f435e/solidity/ecdsa/deploy/09_transfer_proxy_admin_ownership.ts#L14).

Reply:
3ac5b1021e381b2e361e40bc4e0c13ad70a03633